### ISSUE TITLE
[Question]: How to open a file directly?

### ISSUE THREAD

**Issue by lucidBrot** [ahrm/sioyek#610]
Goal: Opening a file directly instead of going through the open file dialog inside sioyek.

## From The Command Line

Often, I already have a terminal open in the current folder, and would like to open a pdf file with sioyek. I have an alias in my `~/.bashrc` so that I can easily launch sioyek:  

```bash
alias sioyek="flatpak run com.github.ahrm.sioyek &>/dev/null"
```

However, I have not yet been able to figure out how to make `sioyek my_file.pdf` open the pdf file directly. Instead, it just launches sioyek with a black screen and then I could open the file manually, by navigating to the correct path again inside the sioyek open dialog. **How can I open a file in a new sioyek instance/window directly from the command line?**

OS: Ubuntu 18.04

## From the File Explorer (Nautilus)
This works. 
I *can* open a file directly from Nautilus. I believe I had manually set up the desktop file at `/var/lib/flatpak/app/com.github.ahrm.sioyek/current/active/files/share/applications/com.github.ahrm.sioyek.desktop` - perhaps sioyek could also set this up itself?

The contents are 
```
[Desktop Entry]
Name=Sioyek
GenericName=PDF Viewer for research papers
Comment=Sioyek is a PDF viewer designed for reading research papers and technical books
Exec=sioyek %f
Icon=com.github.ahrm.sioyek
Terminal=false
Type=Application
Keywords=pdf;viewer;research;
Categories=Office;Viewer;
MimeType=application/pdf;
StartupNotify=false
X-Desktop-File-Install-Version=0.26
```
and this works: The file is opened in sioyek when I right-click it in nautilus and say "open with other application > Sioyek". Which is interesting, considering that the command used is the same.

**Comment by ahrm** [comment]
> not yet been able to figure out how to make `sioyek my_file.pdf`

This should just work. If it doesn't, it is a flatpak-specific issue. You should contact the maintainer of the flatpak package (or just use the appimages in this repository).

**Comment by lucidBrot** [comment]
I got around to testing that now and can confirm. With the AppImage this is no issue, flatpak was to blame. Initially I only used flatpak because building from source didn't immediately work - but the AppImage is definitely a better alternative that I didn't appreciate before.

For future reference: To switch from flatpak to the AppImage I had to make a new `.desktop` file, and copy over my databases like this:
```bash
cp /home/generic/.var/app/com.github.ahrm.sioyek/config/.local/share/Sioyek/local.db /home/generic/.local/share/Sioyek/
cp /home/generic/.var/app/com.github.ahrm.sioyek/config/.local/share/Sioyek/shared.db /home/generic/.local/share/Sioyek/
```

Now it works as expected.
Thanks again!
